package.json

```diff
@@ -53,7 +53,10 @@
     "url-loader": "^0.5.7",
     "urthecast": "^1.0.0",
     "webpack": "^1.12.14",
-    "worker-loader": "^0.7.0"
+    "worker-loader": "^0.7.0",
+    "pbf": "^1.3.2",
+    "vector-tile": "1.2.0",
+    "point-in-polygon": "1.0.0"
```

Comment:
Err yeah he's making things difficult there isn't he.  To be super thorough, you could open a PR into his repo with the license details.  Just copy/paste the standard MIT license text, his name is in his GitHub profile.  Then copy the same text into our LICENSE.md.